### PR TITLE
Bugz 1395/persistent emoji after shutdown

### PR DESCRIPTION
**Pull request by MiladNazeri**
https://highfidelity.atlassian.net/browse/BUGZ-1395

### CODE REVIEW

File: scripts/simplifiedUI/simplifiedEmote/emojiApp/simplifiedEmoji.js [hunk 1]
```diff
@@ -167,6 +167,16 @@ function onScaleChanged() {
 }
 
 
+function onAddingWearable(id) {
+    var props = Entities.getEntityProperties(id, ["name"]);
+    if (props.name.toLowerCase().indexOf("avimoji") > -1) {
+        Entities.deleteEntity(id);
+    } else {
```
Comment (zfox23) [inline scripts/simplifiedUI/simplifiedEmote/emojiApp/simplifiedEmoji.js]: why do you need this whole else block?

File: scripts/simplifiedUI/simplifiedEmote/emojiApp/simplifiedEmoji.js [hunk 2]
```diff
@@ -219,7 +235,8 @@ function createEmoji(emojiFilename) {
         "billboardMode": "full",
         "ignorePickIntersection": true,
         "alpha": 1,
-        "grab": { "grabbable": false }
+        "grab": { "grabbable": false },
+        "userData": JSON.stringify({ timestamp: Date.now() })
```
Comment (zfox23) [inline scripts/simplifiedUI/simplifiedEmote/emojiApp/simplifiedEmoji.js]: you don't need this timestamp if you're using the other `firstEmojiMadeOnStartup` method

File: scripts/simplifiedUI/simplifiedEmote/emojiApp/simplifiedEmoji.js [hunk 3]
```diff
@@ -179,7 +189,13 @@ function onScaleChanged() {
 
 
 // what happens when we need to add an emoji over a user
+var firstEmojiMadeOnStartup = false;
 function addEmoji(emojiFilename) {
+    if (!firstEmojiMadeOnStartup) {
```
Comment (zfox23) [inline scripts/simplifiedUI/simplifiedEmote/emojiApp/simplifiedEmoji.js]: this is risky, but if it works in MOST cases experimentally, i'm ok with this method.

it is feasible that a user could put an emoji up before the old one is deleted.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21190/HighFidelity-Beta-PR16148-b352d53.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21190/HighFidelity-Beta-Interface-PR16148-b352d53.dmg

**Pull request merged**